libs/api/transaction/data-access/src/lib/api-transaction-data-access.service.ts

```diff
@@ -211,8 +212,20 @@ export class ApiTransactionDataAccessService implements OnModuleInit {
       }
     }
 
+    const { errors, signature, solanaStart, status } = appTransaction
+
     // Return object
-    return this.updateAppTransaction(created.id, { webhookEventEnd })
+    return this.updateAppTransaction(created.id, {
+      webhookEventEnd,
+      errors,
+      mint: mint.mint.address,
+      feePayer,
+      signature,
+      solanaCommitted: new Date(),
```

Comment:
The issue here is that this happens at the end of the whole method call. `solanaComitted` should really be set right after we got its confirmation, not after sending webhooks etc.

I think the code as it was, was good. What was the need to add way more data into this `updateTransaction`?

If it is to handle failed Solana transactions, we probably want to return in the `catch` case right after making the Solana transaction.